Automattic/themes
# Skip loading the Learning Mode styles when Sensei is not available

Partially solve [Review spacings and alignment on the course theme#6834](https://github.com/Automattic/sensei/issues/6834)

#### Changes proposed in this Pull Request:
* Skip the learning mode style when sensei is not available
* Add a temporary hook to skip the loading when Sensei is available and able to provide a new learning mode style.

#### Testing instructions
* Go to this branch
* Disable the sensei-lms plugin
* Go to the home page
* Check if the `learning-mode.css` is not loaded
* Enable the sensei-lms plugin
* Check if the `learning-mode.css` is loaded.

### Context
The sensei team is delegating the responsibility to load the learning mode style to the plugin, instead of the handle it on the theme. It will help us to avoid coupling between the theme version and the plugin version.
